nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -235,6 +235,11 @@ language governing permissions and limitations under the License. -->
             <artifactId>json-utils</artifactId>
             <version>0.0.20</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.calcite</groupId>
+            <artifactId>calcite-example-csv</artifactId>
+            <version>1.7.0</version>
```

Comment:
@trixpan yes certainly. I took over this ticket but unfortunately haven't had a chance to work on it much until recently. A week or two ago, I picked it up again and plan to have this into the 1.2.0 version of NiFi. I have updated to the latest version in my local branch. Thanks for calling this out!